### DISCUSSION TITLE
No proteins found in pg_matrix.tsv file

### DISCUSSION THREAD

**Discussion by ABRai11** [vdemichev/DiaNN#972] [Q&A] [answered]
Hi Vadim,

I am searching a plant DIA data in DIA-NN. I couldn't get any proteins in the pg_matrix file. However, there are proteins listed in the report.tsv file. Can you let me know what may be the reason?

Thanks.
Akhila

**Comment by vdemichev** [comment, marked as answer]
Hi Akhila,

Means either no proper protein annotation or nothing passed the FDR filters.
The main report can help see which one it is.

Best,
Vadim

**Comment by ABRai11** [comment]
Hi Vedim,
[DIA_report.zip](https://github.com/vdemichev/DiaNN/files/14735017/DIA_report.zip)

Thank you for the reply. I have uploaded the results here. Could you please check?

**Reply by vdemichev** [reply]
OK, here just nothing passes the FDR filter

**Reply by ABRai11** [reply]
So we cannot use this data?

**Reply by vdemichev** [reply]
Just use the main report and filter proteins not at 1% but at a more relaxed threshold

**Reply by ABRai11** [reply]
I can search with 5% FDR right?

**Reply by vdemichev** [reply]
No, you already have the report that contains all the information you need. Just don't filter it further or filter at 5% protein FDR

**Reply by ABRai11** [reply]
Ok, then is the pg. quantity the abundance value?

**Reply by vdemichev** [reply]
PG.MaxLFQ

**Reply by ABRai11** [reply]
Ok, thanks. Btw may I know the difference.

**Reply by vdemichev** [reply]
PG.MaxLFQ is calculated using the MaxLFQ algorithm, please see the 2014 paper by the Cox lab on this

**Reply by ABRai11** [reply]
Ok, Sure. Thank you so much for clarifying.
